### ISSUE TITLE
eventspy plugin breaking "Run As Maven" when eclipse installed in path with spaces

### ISSUE THREAD

**Issue by kdvolder** [ifedorenko/com.ifedorenko.m2e.mavendev#7]
See: https://issuetracker.springsource.com/browse/STS-4245

This is long... so here's a summary:

The eventspy plugin is adding some -Dmaven.ext.class.path argument to the launch but does not properly escape spaces on Windows, this causes the launch to fail with a weird error.

The problem apears to be specific to Windows. I have tried to reproduce it on Linux as well and couldn't. This makes sense since argument escaping works differently on Windows than on Mac Os / Linux.

**Comment by ifedorenko** [comment]
Dup of https://github.com/ifedorenko/com.ifedorenko.m2e.sourcelookup/pull/16

I'll merge the pull request when I get some free time... but, honestly, installing software into paths with spaces is asking for troubles. Just install Eclipse in a normal directory and move on.

**Comment by ifedorenko** [comment]
Forgot to mention. You can disable `Dynamic Sources Lookup` in Maven launch configuration `Launch Extensions` tab.

**Comment by kdvolder** [comment]
> Just install Eclipse in a normal directory and move on.

I agree, kind of, but it doesn't stop our users from doing this. Not to mention that Windows often creates and installs things in directories with spaces in their names by default, like say 'Program Files' or 'My Documents' or stuff like that.

**Comment by kdvolder** [comment]
This could be a duplicate, but I'm unconvinced. Actually the bug I reproduced involved a *different* plugin than the one that is being patched by that pull request. As far as I can tell this bug has nothing to do with the dynamic source lookup feature.

Anyhoo, you know best, I've done my due dilligence following this up as far as I reasonable feel like I should, whether you choose to look into it further is not my decision.

It looks like Martin has adressed the issue on our end by removing mavendev plugin from our distribution.

**Comment by ifedorenko** [comment]
Maybe you are right, could be two similar problems.
